creusot/src/translation/fmir.rs

```diff
@@ -54,7 +54,7 @@ pub enum Statement<'tcx> {
 #[derive(Clone, Debug)]
 pub enum RValue<'tcx> {
     Ghost(Term<'tcx>),
-    Borrow(Place<'tcx>),
+    Borrow(Place<'tcx>, Span),
```

Comment:
Can you explain the purpose of these specific spans? I can be sold on special casing these, but if possible it would be nice to recover the span from a surrounding `Expr`, `Statement` or `Terminator`.